src/glb-director-xdp/bpf/bpf_helpers.h

```diff
@@ -0,0 +1,63 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
```

Comment:
Annoyingly, not packaged as part of libbpf's packaging. https://github.com/libbpf/libbpf/blob/master/src/bpf_helpers.h